packages/app-builder/public/locales/fr/cases.json

```diff
@@ -7,8 +7,6 @@
   "case.file.added_date": "Ajouté le",
   "case.file.download": "Télécharger",
   "case.file.downloading": "Téléchargement...",
-  "case.file.errors.downloading_decisions_link.auth_error": "Une erreur d'authentification s'est produite lors de la génération du lien de téléchargement. \nReconnectez vous avant de réessayer.",
-  "case.file.errors.downloading_decisions_link.unknown": "Une erreur inconnue s'est produite lors de la génération du lien de téléchargement. \nVeuillez réessayer plus tard.",
```

Comment:
Probably an error, you need them as in the en trad

Reply:
indeed, my bad